### ISSUE TITLE
FSTStringSerializer never call in 2.56

### ISSUE THREAD

**Issue by fihu90** [RuedigerMoeller/fast-serialization#258]
Hi

We want to override the behaviour of FSTStringSerializer.java
But we noticed that it is never called - we checked FSTObjectOutput and there String (line 384) along with Long, Integer and boolean is treated completely separately and is not entering the standard path of serializers. Why is it necessary and is there a way to write our own custom serializer for those types ?

**Comment by RuedigerMoeller** [comment]
yeah there is a shortcut for string serialization as its one of the most frequently used types. This is done for performance reasons.

**Comment by RuedigerMoeller** [comment]
You probably need to branch as hurting performance for many users to support a rare edge case does not make sense. Maybe you could introduce a static boolean switching in between "intrinsic" and regular string serialization. An additional boolean check per string might be acceptable regarding performance
